Formula/austin.rb

```diff
@@ -0,0 +1,25 @@
+class Austin < Formula
+  desc "Python frame stack sampler for CPython"
+  homepage "https://github.com/P403n1x87/austin"
+
+  url "https://github.com/P403n1x87/austin/archive/v1.0.1.tar.gz"
+  sha256 "f86804f777f2f466ddede5d530d3ca67582b2a1467d000662d81272d6e9c5639"
+
+  head "https://github.com/P403n1x87/austin.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+
+  def install
+    system "autoreconf", "--install"
+    system "./configure", "--prefix=#{prefix}"
+    system "make"
+    system "make", "install"
+    mkdir man1.to_s
+    cp "debian/austin.1", man1.to_s
+  end
+
+  test do
+    system "austin", "-V"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.